src/content/en/updates/2018/07/page-lifecycle-api.md

```diff
@@ -603,6 +610,26 @@ window.addEventListener('pagehide', (event) => {
     logStateChange('terminated');
   }
 }, {capture: true});
+
+// Safari does not reliably fire the `pagehide` or `visibilitychange`
+// events when closing a tab, so we have to use `beforeunload` with a
+// timeout to check whether the default action was prevented.
+// - https://bugs.webkit.org/show_bug.cgi?id=151610
+// - https://bugs.webkit.org/show_bug.cgi?id=151234
+// NOTE: we only add this to Safari because adding it to Firefox would
+// prevent the page from being eligible for bfcache.
```

Comment:
Again, I think it would be best to just list this in the "Managing cross-browsers differences" section. If I included all the compat fixes I had in this code sample, it would just end up being the entire contents of the [page-lifecycle](https://github.com/GoogleChromeLabs/page-lifecycle/blob/0.1.1/src/Lifecycle.mjs) library.

Reply:
Done.